tools/cmake/cmake_deps.toml

```diff
@@ -56,6 +56,20 @@ excludes = [
 deps = [
   "executorch",
   "executorch_core",
+  "extension_threadpool",
+  "kernels_util_all_deps",
+]
+
+[targets.kernels_util_all_deps]
+buck_targets = [
+  "//kernels/portable/cpu/util:all_deps",
```

Comment:
if its used by other kernels can we move it out of portable/

Reply:
I tried. So many things can be broken, so I wouldn't bother right now.